chromadb/utils/embedding_functions.py

```diff
@@ -643,6 +645,53 @@ def __call__(self, input: Union[Documents, Images]) -> Embeddings:
         return embeddings
 
 
+class RoboflowEmbeddingFunction(EmbeddingFunction[Images]):
```

Comment:
The type of this embedding function should have a type signature similar to Chroma's own `OpenCLIP` implementation, i.e. `EmbeddingFunction[Union[Documents, Images]]` 

PTAL here for reference:

https://github.com/chroma-core/chroma/blob/86b749ad42457c34044274a30a8ac0c6527d8469/chromadb/utils/embedding_functions.py#L590